### PR TITLE
chore(tabs): add size for tabs right mask

### PR DESCRIPTION
**Pull request by jinlee93**
### Summary:
- noticed right mask for tabs was not being styled correctly, adds size to show mask
image

### Test Plan:
- right mask exists in deployed storybook

### CODE REVIEW

**Comment by dierat** [comment]
Do we have a chromatic test covering this case?

File: src/components/Tabs/Tabs.module.css [hunk 1]
```diff
@@ -34,7 +34,11 @@
 }
 
 .tabs--scrollable-right {
-  -webkit-mask-image: -webkit-linear-gradient(right, transparent);
+  -webkit-mask-image: -webkit-linear-gradient(
+    right,
+    transparent,
+    white var(--eds-size-8)
```
Comment (dierat) [inline src/components/Tabs/Tabs.module.css:40]: Could you help me explain what this changes exactly? I can't tell the difference in Chrome.

**Comment by jinlee93** [comment]
> Do we have a chromatic test that catches the scrollable styling?

We do not, I can add one

File: src/components/Tabs/Tabs.module.css (hunk 1 above)
Comment (jinlee93) [reply src/components/Tabs/Tabs.module.css:40]: Re introduces the right masking when there is no more left content but there is more right content (also in chrome):

Currently:
Screen Shot 2022-10-12 at 12 52 49 PM

This change:
Screen Shot 2022-10-12 at 12 52 47 PM

Comment (dierat) [reply src/components/Tabs/Tabs.module.css:40]: Awesome, thanks!

**Review by dierat — APPROVED** [review]

**Comment by jinlee93** [comment]
@dierat Added some stories for testing the masking, think there is value in keeping these?

**Comment by dierat** [comment]
> @dierat Added some stories for testing the masking, think there is value in keeping these as they don't work when going from the default story to the scroll stories since the play function runs after the viewport change?

But it works in chromatic? I'm fine with keeping them if they work in chromatic 👍

**Comment by jinlee93** [comment]
> But it works in chromatic? I'm fine with keeping them if they work in chromatic 👍

Sounds good to me

**Pull request merged**